### PR TITLE
Feature/naming fix

### PR DESCRIPTION
**Pull request by piotrjaromin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,13 +5,20 @@ A group of Lambda functions for:
 * auto-subscribe new log groups to the aforementioned function so you don't have to subscribe them manually
 * auto-updates the retention policy of new log groups to 7 days (configurable)
 
+as for autosubscribe untion it does not work for existing log groups so you should run scripts from `process_all` directory if adding logs to existing resources.
```
Comment (kgruszka) [inline README.md]: untion -> 'action' ?

File: README.md [hunk 2]
```diff
@@ -5,13 +5,20 @@ A group of Lambda functions for:
...
 
 ## Deployment with code pipeline
 
+For now this is not deployed on den and stage because of qouta limits on logz.io
```
Comment (kgruszka) [inline README.md]: den -> 'dev' ?

**Review by kgruszka — APPROVED** [review]
looks good

File: README.md (hunk 2 above)
Comment (piotrjaromin) [reply README.md]: fixed

File: README.md (hunk 1 above)
Comment (piotrjaromin) [reply README.md]: fixed

**Pull request merged**